lte/gateway/c/oai/tasks/sgw/spgw_state.cpp

```diff
@@ -42,6 +42,8 @@ spgw_state_t* get_spgw_state(bool read_from_db) {
 }
 
 hash_table_ts_t* get_spgw_ue_state() {
+  OAILOG_DEBUG(
```

Comment:
@talkhasib , The `itti_desc` variable is not globally available, so this is not straightforward at the moment. In future PR, we can add an API to expose the thread_id (given the task name) and use it to print a warning here.